.vscode/settings.json

```diff
@@ -14,5 +14,7 @@
   "typescript.preferences.autoImportFileExcludePatterns": [
     "next/router.d.ts",
     "next/dist/client/router.d.ts"
-  ]
+  ],
+  "editor.detectIndentation": false,
+  "editor.tabSize": 2
```

Comment:
I think there is a way to make this your personal setting instead of editing the project `settings.json`

Reply:
I use `tabSize` 4 and `detectIndentation` false for most other projects. Is there a way to modify workspace settings without changing `settings.json`?

Reply:
can this go in your user settings instead? the only rules that should be in the project `settings.json`should be project-specific (ie linting + workspace configuration)

Reply:
Fair enough. I'll take it out and just change the settings on my end